### ISSUE TITLE
Quest doesn't complete / Errors in Console

### ISSUE THREAD

**Issue by PayWithBitcoin** [PikaMug/Quests#1970]
### What is your Quests version / build number (do *not* say "latest")?

Version 4.4.3-b343
(Paper-283 1.18.2)

### Describe the problem and how to replicate it.

My quest "Farmer" doesn't complete even if you have all requirements of that quest finished. 
If someone opens my Quest menu or starts that Quest the errors listet below get spammed in the console.
The error also occured with other plants like beetroots and cocoa.

### Any console errors to share?

```shell
[12:42:04] [Server thread/INFO]: [Quests] 88de6c3c-5e48-4bb7-9c00-00ad98d3a324 started quest Farmer
[12:42:04] [Server thread/WARN]: java.lang.IllegalArgumentException: POTATOES material could not be queried!
[12:42:04] [Server thread/WARN]: 	at Quests-4.4.3.jar//me.blackvein.quests.libs.localelib.LocaleManager.queryMaterial(LocaleManager.java:384)
[12:42:04] [Server thread/WARN]: 	at Quests-4.4.3.jar//me.blackvein.quests.libs.localelib.LocaleManager.sendMessage(LocaleManager.java:125)
[12:42:04] [Server thread/WARN]: 	at Quests-4.4.3.jar//me.blackvein.quests.libs.localelib.LocaleManager.sendMessage(LocaleManager.java:164)
[12:42:04] [Server thread/WARN]: 	at Quests-4.4.3.jar//me.blackvein.quests.Quests.showObjectives(Quests.java:1118)
[12:42:04] [Server thread/WARN]: 	at Quests-4.4.3.jar//me.blackvein.quests.Quester.takeQuest(Quester.java:784)
[12:42:04] [Server thread/WARN]: 	at Quests-4.4.3.jar//me.blackvein.quests.listeners.PlayerListener.onInventoryClickEvent(PlayerListener.java:133)
[12:42:04] [Server thread/WARN]: 	at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor804.execute(Unknown Source)
[12:42:04] [Server thread/WARN]: 	at org.bukkit.plugin.EventExecutor.lambda$create$1(EventExecutor.java:75)
[12:42:04] [Server thread/WARN]: 	at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:80)
[12:42:04] [Server thread/WARN]: 	at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70)
[12:42:04] [Server thread/WARN]: 	at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:629)
[12:42:04] [Server thread/WARN]: 	at net.minecraft.server.network.PlayerConnection.a(PlayerConnection.java:2860)
[12:42:04] [Server thread/WARN]: 	at net.minecraft.network.protocol.game.PacketPlayInWindowClick.a(PacketPlayInWindowClick.java:58)
[12:42:04] [Server thread/WARN]: 	at net.minecraft.network.protocol.game.PacketPlayInWindowClick.a(PacketPlayInWindowClick.java:13)
[12:42:04] [Server thread/WARN]: 	at net.minecraft.network.protocol.PlayerConnectionUtils.lambda$ensureRunningOnSameThread$1(PlayerConnectionUtils.java:51)
[12:42:04] [Server thread/WARN]: 	at net.minecraft.server.TickTask.run(TickTask.java:18)
[12:42:04] [Server thread/WARN]: 	at net.minecraft.util.thread.IAsyncTaskHandler.d(IAsyncTaskHandler.java:153)
[12:42:04] [Server thread/WARN]: 	at net.minecraft.util.thread.IAsyncTaskHandlerReentrant.d(IAsyncTaskHandlerReentrant.java:23)
[12:42:04] [Server thread/WARN]: 	at net.minecraft.server.MinecraftServer.b(MinecraftServer.java:1399)
[12:42:04] [Server thread/WARN]: 	at net.minecraft.server.MinecraftServer.d(MinecraftServer.java:188)
[12:42:04] [Server thread/WARN]: 	at net.minecraft.util.thread.IAsyncTaskHandler.y(IAsyncTaskHandler.java:126)
[12:42:04] [Server thread/WARN]: 	at net.minecraft.server.MinecraftServer.be(MinecraftServer.java:1376)
[12:42:04] [Server thread/WARN]: 	at net.minecraft.server.MinecraftServer.y(MinecraftServer.java:1369)
[12:42:04] [Server thread/WARN]: 	at net.minecraft.util.thread.IAsyncTaskHandler.bo(IAsyncTaskHandler.java:114)
[12:42:04] [Server thread/WARN]: 	at net.minecraft.server.MinecraftServer.a(MinecraftServer.java:1503)
[12:42:04] [Server thread/WARN]: 	at net.minecraft.server.MinecraftServer.w(MinecraftServer.java:1225)
[12:42:04] [Server thread/WARN]: 	at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:315)
[12:42:04] [Server thread/WARN]: 	at java.base/java.lang.Thread.run(Thread.java:833)
[12:42:04] [Server thread/ERROR]: [LocaleLib] Unable to query Material: POTATOES
[12:42:04] [Server thread/WARN]: java.lang.IllegalArgumentException: CARROTS material could not be queried!
[12:42:04] [Server thread/WARN]: 	at Quests-4.4.3.jar//me.blackvein.quests.libs.localelib.LocaleManager.queryMaterial(LocaleManager.java:384)
[12:42:04] [Server thread/WARN]: 	at Quests-4.4.3.jar//me.blackvein.quests.libs.localelib.LocaleManager.sendMessage(LocaleManager.java:125)
[12:42:04] [Server thread/WARN]: 	at Quests-4.4.3.jar//me.blackvein.quests.libs.localelib.LocaleManager.sendMessage(LocaleManager.java:164)
[12:42:04] [Server thread/WARN]: 	at Quests-4.4.3.jar//me.blackvein.quests.Quests.showObjectives(Quests.java:1118)
[12:42:04] [Server thread/WARN]: 	at Quests-4.4.3.jar//me.blackvein.quests.Quester.takeQuest(Quester.java:784)
[12:42:04] [Server thread/WARN]: 	at Quests-4.4.3.jar//me.blackvein.quests.listeners.PlayerListener.onInventoryClickEvent(PlayerListener.java:133)
[12:42:04] [Server thread/WARN]: 	at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor804.execute(Unknown Source)
[12:42:04] [Server thread/WARN]: 	at org.bukkit.plugin.EventExecutor.lambda$create$1(EventExecutor.java:75)
[12:42:04] [Server thread/WARN]: 	at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:80)
[12:42:04] [Server thread/WARN]: 	at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70)
[12:42:04] [Server thread/WARN]: 	at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:629)
[12:42:04] [Server thread/WARN]: 	at net.minecraft.server.network.PlayerConnection.a(PlayerConnection.java:2860)
[12:42:04] [Server thread/WARN]: 	at net.minecraft.network.protocol.game.PacketPlayInWindowClick.a(PacketPlayInWindowClick.java:58)
[12:42:04] [Server thread/WARN]: 	at net.minecraft.network.protocol.game.PacketPlayInWindowClick.a(PacketPlayInWindowClick.java:13)
[12:42:04] [Server thread/WARN]: 	at net.minecraft.network.protocol.PlayerConnectionUtils.lambda$ensureRunningOnSameThread$1(PlayerConnectionUtils.java:51)
[12:42:04] [Server thread/WARN]: 	at net.minecraft.server.TickTask.run(TickTask.java:18)
[12:42:04] [Server thread/WARN]: 	at net.minecraft.util.thread.IAsyncTaskHandler.d(IAsyncTaskHandler.java:153)
[12:42:04] [Server thread/WARN]: 	at net.minecraft.util.thread.IAsyncTaskHandlerReentrant.d(IAsyncTaskHandlerReentrant.java:23)
[12:42:04] [Server thread/WARN]: 	at net.minecraft.server.MinecraftServer.b(MinecraftServer.java:1399)
[12:42:04] [Server thread/WARN]: 	at net.minecraft.server.MinecraftServer.d(MinecraftServer.java:188)
[12:42:04] [Server thread/WARN]: 	at net.minecraft.util.thread.IAsyncTaskHandler.y(IAsyncTaskHandler.java:126)
[12:42:04] [Server thread/WARN]: 	at net.minecraft.server.MinecraftServer.be(MinecraftServer.java:1376)
[12:42:04] [Server thread/WARN]: 	at net.minecraft.server.MinecraftServer.y(MinecraftServer.java:1369)
[12:42:04] [Server thread/WARN]: 	at net.minecraft.util.thread.IAsyncTaskHandler.bo(IAsyncTaskHandler.java:114)
[12:42:04] [Server thread/WARN]: 	at net.minecraft.server.MinecraftServer.a(MinecraftServer.java:1503)
[12:42:04] [Server thread/WARN]: 	at net.minecraft.server.MinecraftServer.w(MinecraftServer.java:1225)
[12:42:04] [Server thread/WARN]: 	at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:315)
[12:42:04] [Server thread/WARN]: 	at java.base/java.lang.Thread.run(Thread.java:833)
[12:42:04] [Server thread/ERROR]: [LocaleLib] Unable to query Material: CARROTS
```

### If applicable, can you provide an example quest from quests.yml?

```yaml
custom20:
    name: Farmer
    ask-message: Los gehts!
    finish-message: Gut gemacht!
    npc-giver-uuid: 3a5255ee-83ea-4730-92d1-20f7685a9a0f
    gui-display:
      ==: org.bukkit.inventory.ItemStack
      v: 2975
      type: HAY_BLOCK
      meta:
        ==: ItemMeta
        meta-type: UNSPECIFIC
        display-name: '{"extra":[{"bold":false,"italic":false,"underlined":false,"strikethrough":false,"obfuscated":false,"color":"dark_purple","text":"Farmer"}],"text":""}'
        lore:
        - '{"extra":[{"text":"Ernte dein Feld!"}],"text":""}'
        ItemFlags:
        - HIDE_ENCHANTS
        - HIDE_ATTRIBUTES
        - HIDE_UNBREAKABLE
        - HIDE_DESTROYS
        - HIDE_PLACED_ON
        - HIDE_POTION_EFFECTS
        - HIDE_DYE
    stages:
      ordered:
        '1':
          break-block-names:
          - WHEAT
          - POTATOES
          - CARROTS
          - SUGAR_CANE
          - MELON
          - PUMPKIN
          break-block-amounts:
          - 4000
          - 2000
          - 2000
          - 1000
          - 500
          - 500
          break-block-durability:
          - 0
          - 0
          - 0
          - 0
          - 0
          - 0
          - 0
          - 0
          cows-to-milk: 100
          items-to-deliver:
          - ==: org.bukkit.inventory.ItemStack
            v: 2975
            type: HAY_BLOCK
            amount: 64
          - ==: org.bukkit.inventory.ItemStack
            v: 2975
            type: MELON
            amount: 64
          - ==: org.bukkit.inventory.ItemStack
            v: 2975
            type: PUMPKIN
            amount: 64
          npc-delivery-uuids:
          - 3a5255ee-83ea-4730-92d1-20f7685a9a0f
          - 3a5255ee-83ea-4730-92d1-20f7685a9a0f
          - 3a5255ee-83ea-4730-92d1-20f7685a9a0f
          delivery-messages:
          - Danke.
          - Danke.
          - Danke.
    rewards:
      items:
      - ==: org.bukkit.inventory.ItemStack
        v: 2975
        type: TURTLE_EGG
        amount: 3
      - ==: org.bukkit.inventory.ItemStack
        v: 2975
        type: NETHERITE_INGOT
      - ==: org.bukkit.inventory.ItemStack
        v: 2975
        type: SHULKER_BOX
      money: 1000
      exp: 1000
    planner:
      override: false
    options:
      allow-commands: true
      allow-quitting: true
      ignore-silk-touch: true
      use-parties-plugin: true
      share-progress-level: 1
      same-quest-only: true
      share-distance: 0.0
      handle-offline-players: false
      ignore-block-replace: true
```

**Comment by PikaMug** [comment]
This is technically an issue with [LocaleLib](https://github.com/PikaMug/LocaleLib), but I'll address it here. It seems Minecraft wants additional data for blocks with age over time. I'll work on a solution for the next release. Thanks!

**Comment by PikaMug** [comment]
Resolved in Quests 4.5.0 release.
